### ISSUE TITLE
Can I set the direct display value

### ISSUE THREAD

**Issue by VX-36998120** [ricreis394/chartjs-card#12]
Can I set the direct display value
As shown in figure
<img width="627" alt="001" src="https://user-images.githubusercontent.com/61342291/192432745-dbf0bea2-3875-46cf-bead-056beb406ad7.png">
<img width="842" alt="002" src="https://user-images.githubusercontent.com/61342291/192432765-ec2fc4e1-86da-47b9-9545-b78f66af0ce7.png">

**Comment by plckr** [comment]
Chartjs is quite flexible, most probably it'll be able to do. I just don't know if it's easy
Did you already search the chartjs docs?

**Comment by siri10086** [comment]
No, first contact with this, I don't know how to search for this feature.

**Comment by siri10086** [comment]
> already

No, first contact with this, I don't know how to search for this feature.

**Comment by VX-36998120** [comment]
The chartjs document has been searched for a long time today, but I haven't found how to set it

**Comment by gamer06** [comment]
Has anyone managed to insert the values at the top of the columns? Thanks
